scripts/env.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# To run as:
+#   curl https://raw.githubusercontent.com/Exabyte-io/api-examples/${GIT_BRANCH}/scripts/env.sh | bash
+
+set -euo pipefail
+
+GIT_BRANCH="${GIT_BRANCH:-dev}"
+REPO_NAME="api-examples"
+VERBOSE="${EXABYTE_API_EXAMPLES_VERBOSE:-}"
+IS_COLAB="${COLAB_JUPYTER_IP:-}"
+NEED_GIT_LFS="${NEED_GIT_LFS:-}"
+
+if [ ! -z "${VERBOSE}" ]; then
+    set -vxeuo pipefail
+    stdout="/dev/stdout"
+else
+    stdout="/dev/null"
+fi
+
+if [ ! -z "${IS_COLAB}" ]; then
+    git clone https://github.com/Exabyte-io/${REPO_NAME}.git --single-branch --branch ${GIT_BRANCH} > ${stdout} 2>&1 || \
+        echo -e "Directory ${REPO_NAME} already exists. Nothing to do." > ${stdout} 2>&1
+
+    cd ${REPO_NAME}
+
+    python -m pip install -r requirements-colab.txt > ${stdout} 2>&1
+    python -m pip install examples/ > ${stdout} 2>&1
```

Comment:
Let's just put `setup.py` to the root

Reply:
Done via https://github.com/Exabyte-io/api-examples/pull/71/commits/e9d9cb18328d885730e05f8f39177e84edbaf66c and subsequent commits to resolve minor importing issues.